internal/os/env.go

```diff
@@ -29,3 +30,25 @@ func MustGetEnv(key, defaultValue string) string {
 	}
 	return value
 }
+
+// MustGetEnvAsBool attempts to parse a bool from a string value retrieved from
+// the specified environment variable. If the environment variable is undefined,
+// the specified default value is returned instead. If the environment variable
+// is defined and its value cannot successfully be parsed as a bool, the
+// function panics.
+func MustGetEnvAsBool(name string, defaultValue bool) bool {
```

Comment:
I believe the `envconfig` library has this build in

https://github.com/kelseyhightower/envconfig

Reply:
I think we can discuss about using `envconfig` in separate issue - Created https://github.com/akuity/kargo/issues/438 for further discussion 🙏